### ISSUE TITLE
Deprecation Alert for theme in ATOM

### ISSUE THREAD

**Issue by AAlvarez90** [JesseLeite/an-old-hope-syntax-atom#30]
![image](https://cloud.githubusercontent.com/assets/7151089/26452319/8061554e-412d-11e7-8f01-99ebd4972536.png)

**Comment by jesseleite** [comment]
@AAlvarez90 Yeah some big things are changing with one-dark-ui as well, with the new github integration, the new tabs, etc.  There'll be some updates coming soon-ish for the theme.  I'll leave this open for now thanks.

**Comment by AAlvarez90** [comment]
@JesseLeite  Thanks to you for this great theme and your contribution!

**Comment by jesseleite** [comment]
Appreciate the kind words.  This theme doesn't always get the updates/love it needs.  Just busy though, I still intend to support it.  PRs welcome, if that's your thing :)

**Comment by AAlvarez90** [comment]
Can you direct me to the documentation of how to create a theme in ATOM? I am not very familiar with the tools it provides to create themes/UI themes.

**Comment by jesseleite** [comment]
No worries :)  I'm not 100% sure about documentation actually.  When I made this theme, I actually forked One Dark and just used their file structure, then used Atom's DOM inspector figure out what to style.

No pressure to contribute though! 🙂

**Comment by AAlvarez90** [comment]
Cool!
